python/pfs/drp/stella/readLineList.py

```diff
@@ -110,25 +197,32 @@ def getLamps(self, metadata):
 
         Returns
         -------
-        lamps : `list` of `str`, or `None`
-            The list of lamps, if the ``restrictByLamps`` configuration option
-            is ``True`` or ``lampList`` is set; otherwise, ``None``.
+        lampInfo : `lsst.pipe.base.Struct`
+           Resultant struct with components:
+            lamps : `set` of `str`, or `None`
+                The list of active lamps from the metadata.
+                If no active lamps can be retrieved from the metadata, and ``assumeSkyIfNoLamps`` is set,
+                then ``[skyLines]`` is returned, otherwise, ``None``.
+            lampElementSet: `set` of `str`, or `None`
```

Comment:
I'd prefer to get rid of the Hungarian notation. Can we just call it `lampElements`? (Throughout this file.)